tests/test_log.py

```diff
@@ -34,7 +34,10 @@
 
 from six.moves import reload_module
 
-
+# We explicitly create a logger which is a child of the tuf hierarchy,
+# instead of using the standard getLogger(__name__) pattern, because the
+# tests are not part of the tuf hierarchy and we are testing functionality
+# of the tuf package explicitly enabled on the tuf hierarchy
 logger = logging.getLogger('tuf.test_log')
```

Comment:
We could avoid this if we move initialising the top-level logger into `tuf/__init__.py`, but as we will still want this module and its API to be able to configure the logger hierarchy it didn't seem particularly beneficial to make that change.